test/test_encryption.py

```diff
@@ -679,6 +693,10 @@ class TestDataKeyDoubleEncryption(EncryptionIntegrationTest):
             'location': 'global',
             'keyRing': 'key-ring-csfle',
             'keyName': 'key-name-csfle'},
+        'kmip': {
+            'keyId': '1',
+            'endpoint': KMIP['endpoint'],  # TODO: is this needed?
```

Comment:
Well, is it?

Reply:
Good catch, the spec actually says to use {} as the kmip key: https://github.com/mongodb/specifications/tree/master/source/client-side-encryption/tests#data-key-and-double-encryption

Fixed.